### PR TITLE
Set event log retention policy

### PR DESCRIPTION
**Pull request by rubenarakelyan**
This commit enforces a retention policy of 2 years for signon event logs. This is achieved with a scheduled job that deletes all log entries older than 2 years every day.

### CODE REVIEW

File: lib/tasks/event_log.rake [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    path = "#{ENV['GOVUK_APP_ROOT'] || Rails.root}/reports"
+    EventLogPresenter.new(min_id, max_id).write_csv(path)
+
+    puts "Report successfully generated inside #{path}"
+  end
+
+  desc "Delete all events in the event log older than 2 years"
+  task delete_logs_older_than_two_years: :environment do
+    delete_count = EventLog.where("created_at < ?", 2.years.ago).delete_all
```
Comment (benlovell) [inline lib/tasks/event_log.rake]: That will delete everything up to two years ago.

Comment (timblair) [reply lib/tasks/event_log.rake]: Yup, that's what we're trying to achieve. From a data retention perspective, we're only keeping audit events for the last two years.

Comment (benlovell) [reply lib/tasks/event_log.rake]: What am I missing? The query says less than 2 years old.

Comment (rubenarakelyan) [reply lib/tasks/event_log.rake]: It's saying anything before 2 years ago...

Comment (timblair) [reply lib/tasks/event_log.rake]: "Where the the created at timestamp is less than (further back in time than) the date two years ago."

Comment (benlovell) [reply lib/tasks/event_log.rake]: Time for me to go home. 🤦‍♂

**Pull request merged**